OpenRA.Game/Map/MapPreview.cs

```diff
@@ -155,6 +156,12 @@ public void UpdateRemoteSearch(MapStatus status, MiniYaml yaml)
 						SpawnPoints = spawns;
 
 						CustomPreview = new Bitmap(new MemoryStream(Convert.FromBase64String(r.minimap)));
+						if (CustomPreview.PixelFormat != PixelFormat.Format32bppArgb)
```

Comment:
If we did the same conversion when loading from disk then we could avoid the discarding.